model/version.go

```diff
@@ -565,9 +566,17 @@ func GetVersionsWithOptions(projectName string, opts GetVersionsOptions) ([]Vers
 		match[bsonutil.GetDottedKeyName(VersionBuildVariantsKey, VersionBuildStatusVariantKey)] = opts.ByBuildVariant
 	}
 
-	if opts.StartAfter > 0 {
-		match[VersionRevisionOrderNumberKey] = bson.M{"$lt": opts.StartAfter}
+	revisionFilter := bson.M{}
+	if opts.Start > 0 {
+		revisionFilter["$lt"] = opts.Start
+		match[VersionRevisionOrderNumberKey] = revisionFilter
```

Comment:
Since we aren't using revisionFilter outside of the if statements or sharing logic, it seems simpler to read to just assign this to {"$lt": opts.Start} and {"$gte": opts.RevisionEnd} directly

Reply:
I set it up this way because if both are defined, we'd first set `match[VersionRevisionOrderNumberKey] = bson.M{"$lt": opts.Start}` and then end up overwriting it with `VersionRevisionOrderNumberKey with bson.M{"$gte": opts.RevisionEnd}`.

Reply:
Oh I see. In that case can we move `match[VersionRevisionOrderNumberKey] = revisionFilter` out of the if statements since it's duplicate code, and this will make it clearer that both parameters are meant to be settable

Reply:
What I was think is that if neither are set, which can happen since they are both optional, it'll end up setting it to `VersionRevisionOrderNumberKey : {}`.

Reply:
Good point, thanks!